src/main/java/thestonedturtle/mahoganyhomes/Home.java

```diff
@@ -144,4 +147,9 @@ static boolean isLadder(final int objID)
 	{
 		return LADDERS.contains(objID);
 	}
+
+	TeleportItem getTeleportItem(Client client)
+	{
+		return this.teleportItems.getClosestTeleportItemOnPlayer(client);
```

Comment:
No need to do add this function, which adds another import to this file, when you can already access `teleportItems` via the `Getter` annotation which allows you to access the `getClosestTeleportItemOnPlayer` function call

```java
currentHome.getTeleportItems().getClosestTeleportItemOnPlayer(client);
```